lib/web_ui/dev/test_runner.dart

```diff
@@ -54,11 +83,59 @@ class TestCommand extends Command<bool> {
   @override
   final String description = 'Run tests.';
 
+  TestTypesRequested testTypesRequested = TestTypesRequested.none;
```

Comment:
I'd use `null` as the default and ensure this is initialized to one of the other values.

Reply:
done.